StockPriceClient/app/src/main/java/drsdrs/stockpriceclient/SyncAdapter.java

```diff
@@ -0,0 +1,114 @@
+package drsdrs.stockpriceclient;
+
+import android.accounts.Account;
+import android.content.AbstractThreadedSyncAdapter;
+import android.content.ContentProviderClient;
+import android.content.ContentResolver;
+import android.content.ContentValues;
+import android.content.Context;
+import android.content.SyncResult;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.google.gson.Gson;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import static android.content.ContentValues.TAG;
+
+/**
+ * Created by ds on 12/8/16.
+ */
+
+public class SyncAdapter extends AbstractThreadedSyncAdapter {
+
+    // Define a variable to contain a content resolver instance
+    ContentResolver mContentResolver;
+    Context mContext;
```

Comment:
No need for a `Context` member variable in this case. In general, saving a reference to a `Context` is something to avoid if possible, since objects that extend `Context` (like `Activity` or `Service`) are large and we don't want to prevent them from being garbage collected by saving an unnecessary reference to them.